cli/internal/ffi/proto/messages.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.0
```

Comment:
Why was this changed in this PR? Why is this file committed at all if it's generated?

Reply:
it seems to be changed by my `rust-analyzer`, reverted.